operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -958,6 +975,20 @@ public void setMaxClusterConcurrentStartup(Integer maxClusterConcurrentStartup)
     this.maxClusterConcurrentStartup = maxClusterConcurrentStartup;
   }
 
+  private int getMaxConcurrentShutdownFor(Cluster cluster) {
```

Comment:
A simpler way to do this would be:

```return Optional.ofNullable(cluster).map(Cluster::getMaxConcurrentShutdown).orElse(getMaxClusterConcurrentShutdown());
```

where getMaxClusterConcurrentShutdown()

Reply:
Fixed.

Reply:
Fixed.